github.js

```diff
@@ -43,10 +43,10 @@ class GitHubClient {
   async hasAlertsEnabled(repos) {
     const enabled = [];
     const disabled = [];
-    repos.forEach((repo) => {
+    repos.forEach(async (repo) => {
```

Comment:
this function is calling an endpoint for each repo, and if it responds with a 403 then that means alerts are not enabled. I added async/await because we were not waiting for the call to finish, so it was always marking every repo as enabled.